Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md

```diff
@@ -0,0 +1,12 @@
+
+#### Scripts
+##### GenerateASMReport
+- Fixed a parsing issue that was causing script to fail.
+- Updated docker image.
```

Comment:
remove the GenerateASMReport from here

Reply:
@adi88d, I am still seeing the issue from https://jira-hq.paloaltonetworks.local/browse/XSUP-20075 and would prefer to keep the changes in there.  I am not sure when that jira ticket is expected to be complete so we would like this as a stop gap solution.

Reply:
@adi88d, i undid the changes to the automation but figured i would update docker image.  Let me know what you think.